src/main/java/org/sunbird/workflow/service/impl/TaxonomyServiceImpl.java

```diff
@@ -149,4 +157,31 @@ private List<HashMap<String, Object>> getUpdateFieldValues(WfStatusEntityV2 stat
         }
         return updateFieldValuesList;
     }
+
+    private StringBuilder constructReadTermApiURI(String term, String category) {
```

Comment:
this method is not called from anywhere ?

Reply:
yes i will remove it is not requried.